### ISSUE TITLE
Is all of the mpi data used?

### ISSUE THREAD

**Issue by gravitychen** [Nicholasli1995/EvoSkeleton#25]
happy spring festival,

I wonder about the details when you evaluate the model on the MPI-inf-3DHP dataset.

did you use all of the  8 subjects && 2 sequences && 14 cameras to inference the model?

**Comment by Nicholasli1995** [comment]
> MPI-inf-3DHP

Hi, we only use the testing set of MPI-INF-3DHP for evaluation. We did not use the training set for fine-tuning. We used the provided 2D keypoints as input to our pre-trained 2D-to-3D lifting model.

**Comment by gravitychen** [comment]
> > MPI-inf-3DHP
> 
> Hi, we only use the testing set of MPI-INF-3DHP for evaluation. We did not use the training set for fine-tuning. We used the provided 2D keypoints as input to our pre-trained 2D-to-3D lifting model.

Nice, so do you normalize MPI testing data using pre-saved stats.npy instead of calculating the mean and std of testing data

**Comment by Nicholasli1995** [comment]
> > > MPI-inf-3DHP
> > 
> > 
> > Hi, we only use the testing set of MPI-INF-3DHP for evaluation. We did not use the training set for fine-tuning. We used the provided 2D keypoints as input to our pre-trained 2D-to-3D lifting model.
> 
> Nice, so do you normalize MPI testing data using pre-saved stats.npy instead of calculating the mean and std of testing data

Yes. We use pre-computed stats.npy to normalize 2D joints of testing data.

**Comment by gravitychen** [comment]
OK I got it.
thank you for your quick reply!

**Comment by ZhangRenkai** [comment]
Hi! Could you please provide the test code or details when evaluating on 3dhp dataset. (About cropping 3dhp images, processing and normalizing the 2D keypoints estimated by HRNet and the 3D GT keypoints coordinates, do they normalized by stats computed from h36m training data or stats computed from 3dhp training data or something else ?)
I normalized 2D GT and 3D GT keypoints using stats computed from h36m training data but the MPJPE error is about 200 mm, and I wonder more details about preprocessing 2D images (scale), processing estimated 2D coordinates and processing 3D coordinates.
And if you have 2D and 3D coordinates of 3dhp dataset like twoDpose_HRN_train.npy and threeDpose_HRN_test.npy in the project. could you please provide it. Thank you so much!!!
